src/realm/obj.cpp

```diff
@@ -866,15 +978,25 @@ void Obj::to_json(std::ostream& out, size_t link_depth, std::map<std::string, st
                     out << "]}";
                 }
                 else {
+
+                    if (output_mode == output_mode_xjson) {
+                        out << "{ \"$embeddedList\": { \"table\": \"" << get_target_table(ck)->get_name()
```

Comment:
Why do we need an embedded list annotation? Is this so your fuzzer tool can verify that it's pointing to the right table?